prog/mutation.go

```diff
@@ -320,7 +320,28 @@ func (t *IntType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []*Ca
 }
 
 func (t *FlagsType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []*Call, retry, preserve bool) {
-	return mutateInt(r, s, arg)
+	if !t.BitMask {
+		return mutateInt(r, s, arg)
+	}
+
+	a := arg.(*ConstArg)
+	switch {
+	case r.nOutOf(1, 10):
+		a.Val = 0
+		for _, val := range t.Vals {
+			if r.bin() {
+				a.Val |= val
+			}
+		}
+	case r.nOutOf(2, 10):
```

Comment:
This can be simplified to nOutOf(1, 5).
Note: all these tests are independent. So r.nOutOf(1, 10) followed by r.nOutOf(2, 10) does not result in 2/10 for the second cases overall. It's 2/10 of the 9/10, so it's 18/100, not a round number anyway ;)